supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from conftest import Fields, TestFields
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        """Test - checks if supplier is saved in DB.
+
+        Args:
+            supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, saved_supplier0):
+        """Test - checks if supplier is not in DB after deletion.
+
+        Args:
+            saved_supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        Supplier.delete_supplier(saved_supplier0)
+        assert saved_supplier0 not in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_filter_by_field(self, saved_supplier0):
+        """Test - checks the Supplier method filter by field.
+
+        The test checks whether the fixtured Supplier - saved_supplier0 -
+        can be found in the returned query set of each attribute and
+        attribute value given to the method "filter by field".
+
+        Args:
+            saved_supplier0: fixture from module conftest.py.
+
+        Returns:
+            None.
+        """
+        for attri, testAttri in zip(Fields, TestFields):
+            assert saved_supplier0 in Supplier.filter_by_field(attri.name,
```

Comment:
Readability is crucial in tests. I'd suggest making the fields visible.

Reply:
I don't think we need it at all. is there another test that uses it? if not, it can be mentioned in the test function itself, explicitly.